components/Account/Memberships/Claim.js

```diff
@@ -215,7 +225,7 @@ class ClaimMembership extends Component {
       serverError,
       values, dirty, errors,
       loading,
-      polling, signInResponse,
+      polling, signInResponse = {},
```

Comment:
I prefer setting default values of state variables in the constructor already, e.g. after line 67:

```
this.state = {
  ...
  dirty: {},
  signInResponse: {}
}